packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js

```diff
@@ -253,16 +253,12 @@ export default function InspectedElementWrapper(_: Props) {
           </div>
         </div>
         {canOpenInEditor && (
-          <Button
-            className={styles.IconButton}
-            onClick={onOpenInEditor}
-            title="Open in editor">
+          <Button onClick={onOpenInEditor} title="Open in editor">
             <ButtonIcon type="editor" />
           </Button>
         )}
         {canToggleError && (
           <Toggle
-            className={styles.IconButton}
```

Comment:
why is this safe to remove?

Reply:
Yeah the class doesn't exist.

https://github.com/facebook/react/blob/c5cfa71948dbf4c2dd9ff64596ab663f8d559cd6/packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.css